pkg/util/openapi/openapi.go

```diff
@@ -228,7 +228,7 @@ func CreateOpenAPIValidator(webServices []*restful.WebService) *Validator {
 	openapispec := LoadOpenAPISpec(webServices)
 	data, err := json.Marshal(openapispec)
 	if err != nil {
-		glog.Fatal(err)
+		log.Fatal(err)
```

Comment:
Please note that `glog.Fatal` calls `os.Exist(2)` result in the application to terminate.
I think we need to call `os.Exist(2)` here, and in other places that that we want to preserve the same behavior.